test/e2e/cloud/nodes.go

```diff
@@ -47,10 +47,11 @@ var _ = SIGDescribe("[Feature:CloudProvider][Disruptive] Nodes", func() {
 		nodeDeleteCandidates := framework.GetReadySchedulableNodesOrDie(c)
 		nodeToDelete := nodeDeleteCandidates.Items[0]
 
-		origNodes := framework.GetReadyNodesIncludingTaintedOrDie(c)
+		origNodes, err := e2enode.GetReadyNodesIncludingTaintedOrDie(c)
```

Comment:
This should be renamed now from `GetReadyNodesIncludingTaintedOrDie` to `GetReadyNodesIncludingTainted`

Reply:
Same comment applies here.